packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -16,17 +16,13 @@ import {
   ShorthandFactory,
   SizeValue,
 } from '../../utils';
+import SplitButtonToggle, { SplitButtonToggleProps } from './SplitButtonToggle';
 import Button, { ButtonProps } from '../Button/Button';
 import MenuButton, { MenuButtonProps } from '../MenuButton/MenuButton';
 import { MenuProps } from '../Menu/Menu';
 import { MenuItemProps } from '../Menu/MenuItem';
 import { PopupProps } from '../Popup/Popup';
 
-export interface SplitButtonSlotClassNames {
```

Comment:
Breaking change?

Reply:
Yep, and the classname is changed. Will add breaking change entry in changelog.